### ISSUE TITLE
Disable page scroll on interaction with vertical rangeslider

### ISSUE THREAD

**Issue by chrissvo** [Stryzhevskyi/rangeSlider#49]
Is there a way to disable the page scroll when interacting with the vertical rangeslider?

**Comment by chrissvo** [comment]
Works beautifully on desktop, but on mobile the interaction is a little clunky due to the page scrolling at the same time.

**Comment by Stryzhevskyi** [comment]
Hi.
Don't see this issue on Google Chrome 76 under Android 9. Could you specify which device you've encountered it on?

**Comment by chrissvo** [comment]
Safari 12, iOS 12.4
https://whatsmybrowser.org/b/Y46QJ5L

**Comment by chrissvo** [comment]
Talking to the duck a bit, found a solution when activating the slider like so:

```js
const slider = document.querySelector('input[type="range"]')

let scrollable = true;=
const listener = e => {
  if (!scrollable) e.preventDefault()
}

document.addEventListener('touchmove', listener, { passive: false })

rangeSlider.create(slider, {
  polyfill: true,     // Boolean, if true, custom markup will be created
  root: document,
  rangeClass: 'rangeSlider',
  disabledClass: 'rangeSlider--disabled',
  fillClass: 'rangeSlider__fill',
  bufferClass: 'rangeSlider__buffer',
  handleClass: 'rangeSlider__handle',
  startEvent: ['mousedown', 'touchstart', 'pointerdown'],
  moveEvent: ['mousemove', 'touchmove', 'pointermove'],
  endEvent: ['mouseup', 'touchend', 'pointerup'],
  vertical: true,     // Boolean, if true slider will be displayed in vertical orientation
  min: 0,             // Number , 0
  max: 150,           // Number, 100
  step: 1,            // Number, 1
  value: 0,           // Number, center of slider
  buffer: null,       // Number, in percent, 0 by default
  stick: null,        // [Number stickTo, Number stickRadius] : use it if handle should stick to stickTo-th value in stickRadius
  borderRadius: 10,   // Number, if you use buffer + border-radius in css for looks good,
  onSlideStart: () => {
    scrollable = false
  },
  onSlideEnd: () => {
    scrollable = true
  }
})
activateRangeslider()
```

**Comment by Stryzhevskyi** [comment]
Check new [build](https://github.com/Stryzhevskyi/rangeSlider/releases/tag/0.4.11), please (already available on NPM, 0.4.11)
